frontend/src/components/Dialogs/Link.js

```diff
@@ -12,14 +12,21 @@ import DialogContent from '@material-ui/core/DialogContent';
 import { UIContext } from 'contexts/ui';
 import { GraphContext } from 'contexts/graph';
 
+import { UI } from './data';
 import useStyles from './styles';
 import DialogCore from './_Dialog';
+import { getValidNodes, validateLink } from './utils/validators';
 
-export const TYPE = 'link';
+/* Constants */
+
+const DEFAULT_STATE = { edge: '', _from: '', _to: '' };
+
+/* Exports */
 
-const defaultState = { edge: '', _from: '', _to: '' };
+export default Dialog;
+export const TYPE = 'link';
```

Comment:
The notion of exporting a `TYPE` variable, is a bit confusing. It makes sense in the context, but if it's going to be exported and used outside, i think it is better to be renamed to `DIALOG_TYPE`.